### DISCUSSION TITLE
Cannot Slew AutoStar Scope

### DISCUSSION THREAD

**Discussion by fdanowski** [Stellarium/stellarium#2771] [Q&A]
I have my telescope connected to my laptop via WiFi (LX200GMD), and I have Stellarium connected to my mount as well (IP 10.0.0.1, port 4030). However, when I select a target and click on Slew... NOTHING. I am using the antenna, and the computer isn't far away from the mount either... Any ideas would be greatly appreciated. Thanks!
![2022-10-22 15_23_15-Greenshot](https://user-images.githubusercontent.com/16124511/197405914-3dc753e6-891d-48b8-8409-ede8ec85a6c0.jpg)
![2022-10-22 15_25_56-Greenshot](https://user-images.githubusercontent.com/16124511/197405923-ac58322c-eb7a-4683-8a8a-146954d872a4.jpg)

**Comment by gzotti** [comment]
We had a little problem in 1.0. https://github.com/Stellarium/stellarium/issues/2713
This caused one OK button to be dead, so it was not able to add or reconfigure a telescope. 
Please try the latest weekly build.

**Reply by fdanowski** [reply]
I just downloaded the latest build, and it is v 1.22.3 but I still cannot slew.

**Reply by gzotti** [reply]
I wrote "weekly build"...

**Comment by alex-w** [comment]
Your configuration will not work in any way because you tried connecting to other Stellarium instance or TelescopeControlServer. Telescope Control plugin doesn't support wifi mounts directly.

**Reply by fdanowski** [reply]
Are you saying that I have to use the ASCOM driver instead of "external software?" When I try to use the ASCOM driver it wants me to choose a Comm Port but it doesn't give me any options in the drop down menu...

**Reply by fdanowski** [reply]
I see what you're saying. I think that once I get the backordered RS232 to USB adapter that I need it'll work fine. I just assumed that since you guys have a mobile app that the desktop version would be at least as advanced regarding WiFi connectivity. It didn't even cross my mind that I might need to tether. Thank You & Be Well!

**Reply by gzotti** [reply]
No, the Stellarium-web and newer mobile versions are totally new developments. Here we keep the desktop version running. If some observer with more modern hardware than was around in 2010 happens to be able to develop support for more recent hardware, we will be happy to receive modernisations.
